### PR TITLE
Add notes about calculating NSTI

### LINKED ISSUE #160: Documentation update: predict metagenomes NSTI values clarification
The current documentation for the -a option in predict metagenomes is a little unclear.  Since the pre-calculated files already have NSTI values available, we should update to explicitly state that you only need to rerun predict_traits.py for custom analysis.

### PR DESCRIPTION
**Pull request by colinbrislawn**
My attempt to close #160 

Work in progress. What other pages should I update?

Also, how do you use `:ref:` to link to files outside of the current folder?

### CODE REVIEW

**Comment by colinbrislawn** [comment]
@wasade Could you help me out with this Travis-CI build? Not sure why this fails... 

>The command "pip install cogent biom-format" failed

**Comment by wasade** [comment]
@GavinHuttley, it looks like there may be some build issues with cogent-1.9 in pypi. The relevant traceback can be found [here](https://travis-ci.org/picrust/picrust/builds/154893653#L532)

@colinbrislawn, can you try revising `.travis.yml` to restrict `cogent==1.5.3`? It should be set as that anyway given the reqs in `setup.py` so it looks like the build is a little out of sync with the stated dependencies.

**Comment by GavinHuttley** [comment]
The cogent issue is [discussed here](https://github.com/pycogent/pycogent/issues/99). Until I've posted a patch, specify `DONT_USE_PYREX=1` in your environment prior to installing.

**Comment by wasade** [comment]
Ah! Thank you and thank you for the quick reply

**Comment by colinbrislawn** [comment]
Thanks guys! Build works great. 

Any feedback on the documentation? Any other pages on which I should discuss NSTI scores?

**Comment by zaneveld** [comment]
Thanks again!

I looked through the most recent commits and all look good. I'll give everyone a moment to also weigh in, but I think this looks ready to merge.

**Comment by wasade** [comment]
:+1:

File: doc/source/tutorials/quality_control.rst [hunk 1]
```diff
@@ -71,6 +71,8 @@ When characterizing PICRUSt’s accuracy, we found that the largest single facto
 
 When comparing NSTI scores to typical bacterial taxonomic cutoffs (e.g. 97% sequence identity), it is worth noting that substitutions/site is equivalent to 1.0 - % identity at *short* phylogenetic distances, but greater than that value at longer phylogenetic distances due to, for example, multiple mutations at the same site that return the nucleotide to a previous state (Supp. Fig. 2 in `Zaneveld et al 2010 <http://nar.oxfordjournals.org/content/suppl/2010/03/02/gkq066.DC1/nar-02618-s-2009-File002.pdf>`_ has a figure relates these two for short distances) 
 
+**NOTE:** NSTI scores have been precalculated for common databases (KEGG Ortholog, COGs, RFAM) and can be easily calculated using the ``-a`` option of :ref:`predict_metagenomes.py <predict_metagenomes>`. See the :`metagenome_prediction` for an example command.
```
Comment (colinbrislawn) [inline doc/source/tutorials/quality_control.rst]: @zaneveld Is this link formatted correctly? I'm pretty new to dynamic documentation...

**Pull request merged**